cmd/cloud/dashboard.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/gosuri/uilive"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-cloud/model"
+)
+
+func init() {
+	dashboardCmd.PersistentFlags().String("server", defaultLocalServerAPI, "The provisioning server whose API will be queried.")
+
+	dashboardCmd.PersistentFlags().Int("refresh-seconds", 10, "The amount of seconds before the dashboard is refreshed with new data.")
+}
+
+var dashboardCmd = &cobra.Command{
+	Use:   "dashboard",
+	Short: "View an auto-refreshing dashboard of all cloud server resources.",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		refreshSeconds, _ := command.Flags().GetInt("refresh-seconds")
+		if refreshSeconds < 1 {
+			return errors.Errorf("refresh seconds (%d) must be set to 1 or higher", refreshSeconds)
+		}
+
+		writer := uilive.New()
+		writer.Start()
+
+		for {
+			tableString := &strings.Builder{}
+			table := tablewriter.NewWriter(tableString)
+			table.SetAlignment(tablewriter.ALIGN_LEFT)
+			table.SetHeader([]string{"TYPE", "TOTAL", "STABLE", "WIP"})
```

Comment:
Maybe something useful for the future would be to split `WIP` column into those that are actually being processed and those that are in a failed state, that are waiting for some action.

That is just an idea for the future, not something for this PR.

Reply:
Yeah, I like this idea for a future PR. Any resources that would fall in the `WIP` column will be listed in detail under the dashboard table so we won't miss that they may have failed, but having this in the table would also be nice.